pkg/processor/trigger/kafka/kafka_test.go

```diff
@@ -100,44 +100,72 @@ func (suite *TestSuite) TestExplicitAckModeWithWorkerAllocationModes() {
 		name                 string
 		explicitAckMode      functionconfig.ExplicitAckMode
 		workerAllocationMode partitionworker.AllocationMode
+		runtime              string
 		expectedFailure      bool
 	}{
 		{
-			name:                 "Disable-Static",
+			name:                 "Python-Disable-Static",
 			explicitAckMode:      functionconfig.ExplicitAckModeDisable,
 			workerAllocationMode: partitionworker.AllocationModeStatic,
+			runtime:              "python",
 			expectedFailure:      false,
 		},
 		{
-			name:                 "Disable-Pool",
+			name:                 "Python-Disable-Pool",
 			explicitAckMode:      functionconfig.ExplicitAckModeDisable,
 			workerAllocationMode: partitionworker.AllocationModePool,
+			runtime:              "python",
 			expectedFailure:      false,
 		},
 		{
-			name:                 "Enable-Static",
+			name:                 "Python-Enable-Static",
 			explicitAckMode:      functionconfig.ExplicitAckModeEnable,
 			workerAllocationMode: partitionworker.AllocationModeStatic,
+			runtime:              "python",
 			expectedFailure:      false,
 		},
 		{
-			name:                 "Enable-Pool",
+			name:                 "Python-Enable-Pool",
 			explicitAckMode:      functionconfig.ExplicitAckModeEnable,
 			workerAllocationMode: partitionworker.AllocationModePool,
+			runtime:              "python3.9",
 			expectedFailure:      true,
 		},
 		{
-			name:                 "ExplicitOnly-Static",
+			name:                 "Python-ExplicitOnly-Static",
 			explicitAckMode:      functionconfig.ExplicitAckModeExplicitOnly,
 			workerAllocationMode: partitionworker.AllocationModeStatic,
+			runtime:              "python3.10",
 			expectedFailure:      false,
 		},
 		{
-			name:                 "ExplicitOnly-Pool",
-			explicitAckMode:      functionconfig.ExplicitAckModeEnable,
+			name:                 "Python-ExplicitOnly-Pool",
+			explicitAckMode:      functionconfig.ExplicitAckModeExplicitOnly,
 			workerAllocationMode: partitionworker.AllocationModePool,
+			runtime:              "python",
 			expectedFailure:      true,
 		},
+		{
+			name:                 "Golang-Enable-Pool",
+			explicitAckMode:      functionconfig.ExplicitAckModeEnable,
+			workerAllocationMode: partitionworker.AllocationModePool,
+			runtime:              "golang",
+			expectedFailure:      false,
+		},
+		{
+			name:                 "Golang-ExplicitOnly-Pool",
+			explicitAckMode:      functionconfig.ExplicitAckModeEnable,
+			workerAllocationMode: partitionworker.AllocationModeStatic,
+			runtime:              "golang",
+			expectedFailure:      false,
```

Comment:
We both missed this one.
Either change the name or the modes, not sure what was intended.

Reply:
@TomerShor changed the mode